services/basic_types.proto

```diff
@@ -1592,3 +1592,20 @@ message StakingInfo {
         int64 staked_node_id = 6;
     }
 }
+
+/**
+ * Info about a contract account's nonce value.
+ * A nonce of a contract is only incremented when that contract creates another contract.
+ */
+message ContractNonceInfo {
+
+    /**
+    * Id of the contract
+    */
+    ContractID contract_id = 1;
+
+    /**
+    * The current value of the contract account's nonce property
+    */
+    int64 nonce = 2;
+}
```

Comment:
Is there a reason this is in basic_types.proto instead of contract_call_local.proto?

Reply:
Checked with @stoqnkpL and we don't think so, will move it to `contract_call_local.proto`.